src/models/gaze17/external-overview.md

```diff
@@ -0,0 +1,63 @@
+# Gazelle (External Overview)
+
+### Left side:
+
+![17" Left Ports](./img/ports-left-17.webp)
+
+### Right side:
+
+![17" Right Ports (3060)](./img/ports-right-17-3060.webp)
+
+### Front side:
+
+![17" Front Ports](./img/ports-front-17.webp)
+
+### Back side:
+
+![17" Back Ports (3060)](./img/ports-back-17-3060.webp)
+
+## Materials:
+
+|Part                              |Material |
+|----------------------------------|---------|
+|LCD back cover (lid)              |Plastic  |
+|LCD front cover (bezel)           |Plastic  |
+|Top case (palm rests/port covers) |Plastic  |
+|Bottom panel                      |Plastic  |
+
+## LED indicators:
+
+The Gazelle has the following LED indicators:
+
+|Icon                                    |Color          |Description                      |
+|----------------------------------------|---------------|---------------------------------|
+|![Power LED](./img/led-power.png)       |Orange         |Powered off, DC power plugged in |
+|                                        |Green          |Powered on                       |
+|                                        |Blinking green |Sleeping (suspended)             |
+|![Battery LED](./img/led-battery.png)   |Orange         |Battery charging                 |
+|                                        |Green          |Battery fully charged            |
+|                                        |Blinking orange|Battery critically low           |
+|![Storage LED](./img/led-storage.png)   |Green          |Storage drive activity           |
+
+## Keyboard shortcuts:
+
+The Gazelle has the following actions available using the Fn and Function keys:
+
+|Key                        |Shortcut|Action                       |
+|---------------------------|--------|-----------------------------|
+|![Fn-F1](./img/fn-f1.png)  |Fn+F1   |Toggle trackpad              |
+|![Fn-F2](./img/fn-f2.png)  |Fn+F2   |Toggle built-in LCD          |
+|![Fn-F3](./img/fn-f3.png)  |Fn+F3   |Mute                         |
+|![Fn-F5](./img/fn-f5.png)  |Fn+F5   |Volume down                  |
+|![Fn-F6](./img/fn-f6.png)  |Fn+F6   |Volume up                    |
+|![Fn-F7](./img/fn-f7.png)  |Fn+F7   |Toggle displays              |
+|![Fn-F8](./img/fn-f8.png)  |Fn+F8   |Screen brightness down       |
+|![Fn-F9](./img/fn-f9.png)  |Fn+F9   |Screen brightness up         |
+|![Fn-F10](./img/fn-f10.png)|Fn+F10  |Toggle webcam                |
+|![Fn-F11](./img/fn-f11.png)|Fn+F11  |Toggle airplane mode         |
+|![Fn-F12](./img/fn-f12.png)|Fn+F12  |Suspend                      |
+|![Fn-F12](./img/fn-dia.jpg)|Fn+`    |Play/Pause                   |
+|![Fn-*](./img/fn-star.png) |Fn+*    |Toggle keyboard backlight    |
+|/                          |Fn+/    |Cycle keyboard color         |
+|-                          |Fn+-    |Decrease keyboard brightness |
+|+                          |Fn++    |Increase keyboard brightness |
```

Comment:
Should we include `Fn`+`1` for fan toggle? Looks like we aren't super consistent about it, but it's probably good to include.

Reply:
Yes, we should include that. Added with d8caecae30da0950f47cea6deb8afabe73767012.